File: 2161.py
N = int(input())

if N == 0:
    print(f"{3.0:.10f}")
else:
    x = 6.0
    for _ in range(1, N):
        x = 6.0 + 1.0 / x

    ans = 3.0 + 1.0 / x
    print(f"{ans:.10f}")

File: 2161.js
const fs = require("fs");
const N = parseInt(fs.readFileSync(0, "utf8").trim());

if (N === 0) {
    console.log("3.0000000000");
} else {
    let x = 6.0;
    for (let i = 1; i < N; i++) {
        x = 6.0 + 1.0 / x;
    }
    let ans = 3.0 + 1.0 / x;
    console.log(ans.toFixed(10));
}
